src/test/java/MenuTest.java

```diff
@@ -0,0 +1,93 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MenuTest {
+
+    private Menu menu;
+    private final PrintStream standardOut = System.out;
```

Comment:
Maybe add a comment here telling students they will learn more about these concepts in a later unit?

Reply:
Added a brief comment